pacific-1/proposals/08-11-2023-mint-schedule.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Mint Param Change Proposal",
+  "description": "Proposal to add Daily Mint Token Release Schedule",
+  "changes": [
+    {
+      "subspace": "mint",
+      "key": "TokenReleaseSchedule",
+      "value": [
+	    {
+	      "token_release_amount": "300000000",
+	      "start_date": "2023-08-15",
+	      "end_date": "2024-08-14"
```

Comment:
Just to double check, is the "end date" inclusive or exclusive?

Reply:
end date is inclusive